src/server/replica.cc

```diff
@@ -635,18 +635,24 @@ error_code Replica::ConsumeRedisStream() {
     }
 
     if (!LastResponseArgs().empty()) {
-      VLOG(2) << "Got command " << absl::CHexEscape(ToSV(LastResponseArgs()[0].GetBuf()))
-              << "\n consumed: " << response->total_read;
-
-      if (LastResponseArgs()[0].GetBuf()[0] == '\r') {
-        for (const auto& arg : LastResponseArgs()) {
-          LOG(INFO) << absl::CHexEscape(ToSV(arg.GetBuf()));
+      string_view cmd = absl::CHexEscape(ToSV(LastResponseArgs()[0].GetBuf()));
+
+      // Valkey and Redis may send MULTI and EXEC as part of their replication commands.
+      // Dragonfly disallows some commands, such as SELECT, inside of MULTI/EXEC, so here we simply
+      // ignore MULTI/EXEC and execute their inner commands individually.
+      if (!absl::EqualsIgnoreCase(cmd, "MULTI") && !absl::EqualsIgnoreCase(cmd, "EXEC")) {
```

Comment:
The only change here is the added `if` (with comments), and the usage of `cmd` in the `VLOG` below